src/Microsoft.OData.Client/Build.NuGet/Microsoft.OData.Client.Nightly.Release.nuspec

```diff
@@ -14,6 +14,7 @@
     <summary>LINQ-enabled client API for issuing OData queries and consuming OData JSON payloads. Supports OData v4 only. </summary>
     <description>LINQ-enabled client API for issuing OData queries and consuming OData JSON payloads. Supports OData v4 only. Targets .NET 4.0, and .NET Portable Lib with support for .NET 4.5, Win Phone 8, Win Phone 8.1, and Win 8. Localized for CHS, CHT, DEU, ESN, FRA, ITA, JPN, KOR and RUS.
 OData .NET library is open source at http://github.com/OData/odata.net</description>
+    <copyright>© Microsoft Corporation.  All rights reserved.</copyright>

```

Comment:
I'm guessing this works as opposed to using the escape sequence, i.e. &#169; ?

Reply:
So I wrote the escape sequence and that was displayed as the character not the sequence. What I typed was: &, then #, then 169, then ;

Reply:
Yes it should work. That is the style used in other places such as Microsoft.OData.StyleCop.nuspec and Microsoft.NETCore.App.nuspec.
As we discussed in Rob's office, we can proceed the merge.